README.md

```diff
@@ -21,7 +21,7 @@
 
 # Visual Studio Application Insights SDK for .NET Web Applications
 
-The code in this repository is the .NET web application SDK for Application Insights. [Application Insights][AILandingPage] is a service that lets you monitor your live application's performance and usage. This SDK sends telemetry to the service. It collects data such as web request timings and success rates, dependency calls, exceptions, and server performance counters. You can also use the SDK to send your own telemetry and add modules to collect logs. You can use this SDK in any .NET web application, hosted either on your own servers or on Microsoft Azure.
+The code in this repository is the .NET web application SDK for Application Insights. [Application Insights][AILandingPage] is to monitor performance and usage of user’s live application.. This SDK sends telemetry to the service. It collects data such as web request timings and success rates, dependency calls, exceptions, and server performance counters. You can use the SDK to send your telemetry and add modules to collect logs. This SDK can be used in any .NET web application hosted on your servers or on Microsoft Azure.
```

Comment:
is to monitor performance and usage of user’s live application..  -- this change does not seem better than the previous one.